conan/tools/cmake/cmakedeps/templates/macros.py

```diff
@@ -88,6 +88,9 @@ def template(self):
                      message(DEBUG "Created target ${_LIB_NAME} ${library_type} IMPORTED")
                      set_target_properties(${_LIB_NAME} PROPERTIES IMPORTED_LOCATION${config_suffix} ${CONAN_FOUND_LIBRARY} IMPORTED_NO_SONAME ${no_soname_mode})
                    endif()
+                   string(REGEX_REPLACE "^_" "" _LOWER_CASE_CONFIG ${config_suffix})
```

Comment:
This is broken:
```
string does not recognize sub-command REGEX_REPLACE
```
Note that support for CMake 3.15 is necessary.

Reply:
Thanks for the feedback, this was a simply typo (underscore instead of space). Should be fixed now, lets see what the next CI run says. The used CMake function should be supported at least since 3.0 (never used 2.x before, probably it even existed there).

Reply:
Looks like at least the CI is now happy :smile: .

Regarding the CMake support I just noticed that the same function [is already used a few lines above](https://github.com/conan-io/conan/pull/16705/files#diff-c4faa1c61f8d99a6ae53496c9eea30c47d48491daa851281e034e60b83b5bb56R57).